cypress/integration/MAT/WebInterface/HarpUser/HarpIdWithMultipleMatUsers.spec.js

```diff
@@ -0,0 +1,32 @@
+import * as oktaLogin from '../../../../support/oktaLogin'
+import * as helper from '../../../../support/helpers'
+import * as matheader from '../../../../pom/MAT/WI/MATheader'
+import * as measurelibrary from "../../../../pom/MAT/WI/MeasureLibrary";
+
+describe('HARP ID: Multiple MAT Users', () => {
+    before('Login', () => {
+        oktaLogin.login()
+    })
+    beforeEach('Preserve Cookies', () => {
+        helper.preserveCookies()
+    })
+    after('Log Out', () => {
+        helper.logout()
+    })
+
```

Comment:
Do we need a special user is this case? I don't think I could use the same users that I am currently using to satisfy this. We might need to setup another user with corresponding environment variables. Also I think we might need to edit the Logout Button as its element definition is dependent on its specific place in the User Dropdown.

Reply:
I did some modification to adapt multiple MAT users scenario. I closing this PR and going to add new one. Thanks!